x-pack/plugins/monitoring/dev_docs/how_to/apm_tracing.md

```diff
@@ -0,0 +1,43 @@
+To get deeper information about the behavior of the stack monitoring application you can configure APM tracing.
```

Comment:
can we specify here: "you can configure APM to send trace data to your own APM server + kibana instance" -- whenever I see these suggestions my mind locks up with "wait, do they mean the data goes to some magic central cluster or that I need to set something up to send these data to"

Reply:
Yeah, I'll re-word. It's sort of both. If you work here or use ESS, you already have an APM server available. On-prem would need to launch their own.

Reply:
Makes sense. I think it helps people understand if we start with the baseline "these are the configs for sending APM data to a server" and then build on that by saying "in some contexts, there are some magic variables you can set which auto sends data to existing APM servers" or something like that. At least that's why it's been confusing to me a lot of times in the past, anecdotally. Thanks!